pkg/users/account-create-dialog.js

```diff
@@ -162,31 +208,59 @@
     return remove_diacritics(result);
 }
 
-export function account_create_dialog(accounts) {
+export function account_create_dialog(accounts, shells) {
     let dlg = null;
     const state = {
+        dialogLoading: true,
         real_name: "",
         user_name: "",
         password: "",
         password_confirm: "",
         locked: false,
         confirm_weak: false,
         change_passw_force: false,
+        base_home_dir: "",
+        home_dir: "",
+        home_dir_dirty: false,
+        shell: "",
+        isShellSelectExpanded: false,
     };
     let errors = { };
 
     let old_password = null;
     let user_name_dirty = false;
 
+    function get_defaults() {
+        return cockpit.spawn(["/usr/sbin/useradd", "-D"], { superuser: "require" })
+                .catch(() => "")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230221-092948-26bd4da0-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)